interface/patient_file/history/history.php

```diff
@@ -58,22 +54,19 @@
 ?>
 
 <?php if (acl_check('patients','med','',array('write','addonly') )) { ?>
-<div>
-    <span class="title"><?php echo htmlspecialchars(xl('Patient History / Lifestyle'),ENT_NOQUOTES); ?></span>
-</div>
-<div id='namecontainer_history' class='namecontainer_history' style='float:left;margin-right:10px'>
-<?php echo htmlspecialchars(xl('for'),ENT_NOQUOTES);?>&nbsp;<span class="title"><a href="../summary/demographics.php" onclick="top.restoreSession()"><?php echo htmlspecialchars(getPatientName($pid),ENT_NOQUOTES) ?></a></span>
+<div class="page-header">
+    <h1><?php echo htmlspecialchars(getPatientName($pid), ENT_NOQUOTES);?> <small><?php echo xl("History & Lifestyle");?></small></h1>
 </div>
 <div>
-    <a href="history_full.php"
-     class="css_button"
-     onclick="top.restoreSession()">
-    <span><?php echo htmlspecialchars(xl("Edit"),ENT_NOQUOTES);?></span>
+<div class="btn-group">
+    <a href="../summary/demographics.php" class="btn btn-default" onclick="top.restoreSession()">
+        <i class="fa fa-chevron-left"></i>&nbsp;&nbsp;<?php echo htmlspecialchars(xl('Back To Patient'),ENT_NOQUOTES);?>
```

Comment:
another option here is to use the btn-transmit in class rather than the chevron-left